### PR TITLE
More misc fixes

### PR DESCRIPTION
**Pull request by JoosuaKoskinen**
#### What are the main changes in this PR?
Closes: #617, #624, #623, #619

- Fixes Slack webhook
- Fixes wp-backup-status ran before first backup
- Removes WP CLI 'seravo updates' from dev and staging envs.
- Fix zone sniff on finnish sites.

### CODE REVIEW

File: src/modules/pages/backups.php [hunk 1]
```diff
@@ -133,7 +133,7 @@ function ( Component $base, Postbox\Postbox $postbox ) {
    * @return Ajax\AjaxResponse
    */
   public static function get_backups_status() {
-    if ( ! \file_exists('/data/backups') || ! \file_exists('/data/backups/data') ) {
+    if ( ! \file_exists('/data/backups') || ! \file_exists('/data/backups/data') || ! \file_exists('/data/backups/exclude.filelist') ) {
```
Comment (Moppa5) [inline src/modules/pages/backups.php:136]: I'm not sure whether the missing `exclude.filelist` should cause any issues? It seems that `wp-backup-status` just outputs that the file is missing. Did you run into problems with this?
```
--> Comparing /data to latest backup...
cat: /data/backups/exclude.filelist: No such file or directory
...
```

**Review by Moppa5 — APPROVED** [review]
Looking good! I had one comment about the `exclude.filelist`, but if you have had issues with it missing then I think we are good to go.

File: src/modules/pages/backups.php (hunk 1 above)
Comment (JoosuaKoskinen) [reply src/modules/pages/backups.php:136]: `exclude.filelist` is created on first `wp-backup`. If it doesn't exists, backups neither do

Comment (Moppa5) [reply src/modules/pages/backups.php:136]: Well alright. Still find it rather excessive as we are already checking here whether the backups exist at all. However, this is no big deal and we can stick with that if you want to.

**Pull request merged**